### PR TITLE
Simple speedometer

### LINKED ISSUE #24: Speedometer
As a user, I want to be able to see the current speed of the car while I drive so that I can control the car properly.
Acceptance Criteria: 

- [x] The app should provide the car’s current speed information.

### PR DESCRIPTION
**Pull request by nk-io**
Text-based speedometer is ready and functional

Closes #24

### CODE REVIEW

File: client/app/src/main/java/group01/smartcar/client/SimpleSpeedometer.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package group01.smartcar.client;
+
+public class SimpleSpeedometer{
+}
```
Comment (JensTheS) [inline client/app/src/main/java/group01/smartcar/client/SimpleSpeedometer.java]: This file is not used, I suggest we remove it.

**Review by JensTheS — CHANGES_REQUESTED** [review]
Great work on the speedometer, it looks clean.
I have a delay (30-60seconds) when I connect to MQTT from our external server: hysm.dev. 
Running the app with a local MQTT gives a good flow and the speedometer responds nicely.
If we want to run MQTT on an external server we need to fix this.

File: client/app/src/main/java/group01/smartcar/client/SimpleSpeedometer.java (hunk 1 above)
Comment (nk-io) [reply client/app/src/main/java/group01/smartcar/client/SimpleSpeedometer.java]: The unnecessary file has been deleted. I switched the MQTT connection to localhost. As you said it'll give better results.

**Comment by gsastraw** [comment]
Looks good to me, running the app on a local MQTT server is a lot more responsive. Good work!

**Review by JensTheS — APPROVED** [review]
Looks better.

**Pull request merged**